.github/workflows/release.yml

```
@@ -60,14 +95,14 @@ jobs:
     with:
-      tag: ${{ inputs.version }}
+      tag: ${{ needs.release.outputs.version }}
     permissions:
       contents: write
 
   update-homebrew:
     needs: wails-build
     uses: ./.github/workflows/update-homebrew.yml
     with:
-      tag: ${{ inputs.version }}
+      tag: ${{ needs.release.outputs.version }}
```

actionlint (1.7.11)

[error] property "release" is not defined in object type {wails-build: {outputs: {}; result: string}}

(expression)